### PR TITLE
Fix interactive execution bug

### LINKED ISSUE #1771: F5 in VSCode - Cannot index into a null array
## General summary of the issue

I have a rather simple scenario where I've enabled Pester 5 in VSCode and I can run pester using Run tests without any problems or when I start it up from the primary file. 

But once I try to press F5 on the file

image

It errors out

```
Cannot index into a null array.
At C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Modules\Pester\5.1.0\Pester.psm1:8471 char:13
+             Invoke-Pester -Path $Path | & $SafeCommands['Out-Null']
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray
```

VSCode settings:

image

## Describe your environment

```
Pester version     : 5.1.0-rc1 C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Modules\Pester\5.1.0\Pester.psm1
PowerShell version : 5.1.19041.610
OS version         : Microsoft Windows NT 10.0.19042.0
```

## Expected Behavior

No error, runs as with pressing a button

## Current Behavior

Erorr

```
Cannot index into a null array.
At C:\Users\przemyslaw.klys\Documents\WindowsPowerShell\Modules\Pester\5.1.0\Pester.psm1:8471 char:13
+             Invoke-Pester -Path $Path | & $SafeCommands['Out-Null']
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : NullArray
```

## Possible Solution?

Nope :-)

### PR DESCRIPTION
**Pull request by fflaten**
## PR Summary

Fixes interactive execution of pester tests (ex. using F5 in VSCode). Bug caused by #1745 which added SafeCommands-reference in a scriptblock where it isn't available

Fix #1771 

## PR Checklist

- [x] PR has meaningful title
- [x] Summary describes changes
- [x] PR is ready to be merged
  - If not, use the arrow next to `Create Pull Request` to mark it as a draft. PR can be marked `Ready for review` when it's ready.
- [x] Tests are added/update *(if required)*
- [ ] Documentation is updated/added *(if required)*

### CODE REVIEW

**Comment by fflaten** [comment]
Not proud of the current test. Possible to get resultobject when using interactive execution?

**Comment by nohwnd** [comment]
Not easily, but you could run it in separate process. Just put it in the "tst\Pester.RSpec.Output.ts.ps1" and check for the line we output when the test finishes successfully.

**Comment by fflaten** [comment]
Yeah that was my backup, but felt misplaced. Guess it's better than a no assert. 🙂

**Comment by nohwnd** [comment]
You can rename the file, to InNewProcess instead of output, and it's fine 👍

**Comment by nohwnd** [comment]
Ping me when this is ready, I will make RC2.  :)

**Comment by fflaten** [comment]
@nohwnd Read for review.

Ended up placing it in a new file since we needed a new helper-method that didn't force `Invoke-Pester`. The output-file was already neat with only output-tests atm.

**Comment by nohwnd** [comment]
> Ended up placing it in a new file since we needed a new helper-method that didn't force `Invoke-Pester`. The output-file was already neat with only output-tests atm.

Pls add a comment about that so we don't accidentally break it and wonder why.

File: src/functions/Describe.ps1 [hunk 1]
```diff
@@ -151,12 +151,12 @@ function Invoke-Interactively ($CommandUsed, $ScriptName, $SessionState, $BoundP
         # but make sure we are invoking it in the caller session state, because
         # paths don't stay attached to session state
         $invokePester =  {
-            param($private:Path)
-            Invoke-Pester -Path $Path | & $SafeCommands['Out-Null']
+            param($private:Path, $Out_Null)
```
Comment (nohwnd) [inline src/functions/Describe.ps1:154]:
```suggestion
            param($private:Path, $private:Out_Null)
```

Comment (nohwnd) [reply src/functions/Describe.ps1:154]: Added private otherwise we would leave it in the user scope.

**Pull request merged**